daemon/oci_linux.go

```diff
@@ -1007,7 +1007,7 @@ func WithUser(c *container.Container) coci.SpecOpts {
 	}
 }
 
-func (daemon *Daemon) createSpec(c *container.Container) (retSpec *specs.Spec, err error) {
+func (daemon *Daemon) createSpec(ctx context.Context, c *container.Container) (retSpec *specs.Spec, err error) {
```

Comment:
There's a `context.Background()` call on line 1055 which could be updated.

Reply:
Thanks! I had it in my experimental branch, but forgot to carry it over here :)